go.mod

```diff
@@ -1,33 +1,40 @@
 module github.com/bitrise-io/bitrise-init
 
-go 1.21
+go 1.22.0
+
+toolchain go1.23.7
```

Comment:
Could you revert these lines? 

We are building the steps on the fly (and the project scanner step relies on this package) which means we cannot really just bump the Go version number without updating the preinstalled Go runtime on the stacks. At the moment the older stacks have Go 1.21 on them.